assets/js/modules/search-console/components/dashboard/SearchFunnelWidgetGA4/AnalyticsStats.js

```diff
@@ -0,0 +1,196 @@
+/**
+ * AnalyticsStats component for SearchFunnelWidgetGA4.
+ *
+ * Site Kit by Google, Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import Data from 'googlesitekit-data';
+import { Grid, Row, Cell } from '../../../../../material-components';
+import { CORE_MODULES } from '../../../../../googlesitekit/modules/datastore/constants';
+import { extractAnalyticsDashboardData } from '../../../../analytics-4/utils';
+import GoogleChart from '../../../../../components/GoogleChart';
+const { useSelect } = Data;
+
+export default function AnalyticsStats( props ) {
```

Comment:
Same here. It's identical to its analog from the original widget. Let's move it to the common folder and update this component to accept different params via props.

Please, review other components as well. If they are almost identical, let's move them to the `common` folder and pass differences via props.

Reply:
The same applies here as well.

cc: @techanvil WDYT?

Reply:
I'd agree with this, it doesn't seem worth adding complexity to the component to make it reusable when we know we're going to delete the UA version of the widget. Creating a new version seems more pragmatic here, IMHO.

Reply:
I think the IB is working on the assumption that the components to be reused would not require any changes.

However, if it's not a significant effort to restructure them to make reusable, I don't have a strong opinion against it.

Worth mentioning though, is that I ran into some similar issues in #6216 where the IB assumed components would be reusable but they were not and I've duplicated in that issue, although the differences might have been more involved than this one. So it might be more consistent now to duplicate here as well. :shrug:

Reply:
I agree with @techanvil. The IB assumes that the child components could be reusable. However, the shape of the GA4 `getReports` data is different than for the UA reports. Introducing couple or three props can make the `AnalyticsStats` and `ActivateAnalyticsCTA` components reusable. However, the `Overview` component has to be separate.

Reply:
That's okay if some components will be duplicated, I have nothing against it. But there are small components that don't need much effort to make them re-usable. Let's work on those and keep harder ones duplicated.